code/game/objects/items/weapons/implants/implant_loyality.dm

```diff
@@ -1,65 +1,65 @@
-/obj/item/weapon/implant/loyalty
-	name = "loyalty implant"
-	desc = "Makes you loyal or such."
+/obj/item/weapon/implant/mindshield
+	name = "mindshield implant"
+	desc = "Protects against brainwashing."
 	origin_tech = "materials=2;biotech=4;programming=4"
 	activated = 0
 
-/obj/item/weapon/implant/loyalty/get_data()
+/obj/item/weapon/implant/mindshield/get_data()
 	var/dat = {"<b>Implant Specifications:</b><BR>
 				<b>Name:</b> Nanotrasen Employee Management Implant<BR>
 				<b>Life:</b> Ten years.<BR>
-				<b>Important Notes:</b> Personnel injected with this device tend to be much more loyal to the company.<BR>
+				<b>Important Notes:</b> Personnel injected with this device are much more resistant to brainwashing.<BR>
 				<HR>
 				<b>Implant Details:</b><BR>
-				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
+				<b>Function:</b> Contains a small pod of nanobots that protects the host's mental functions from manipulation.<BR>
 				<b>Special Features:</b> Will prevent and cure most forms of brainwashing.<BR>
 				<b>Integrity:</b> Implant will last so long as the nanobots are inside the bloodstream."}
 	return dat
 
 
-/obj/item/weapon/implant/loyalty/implant(mob/target)
+/obj/item/weapon/implant/mindshield/implant(mob/target)
 	if(..())
 		if((target.mind in (ticker.mode.head_revolutionaries | ticker.mode.get_gang_bosses())) || is_shadow_or_thrall(target))
-			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
+			target.visible_message("<span class='warning'>[target] seems to resist the implant!</span>", "<span class='warning'>You feel something try to heal your mind, but you resist it!</span>")
 			removed(target, 1)
 			qdel(src)
 			return -1
 		if(target.mind in ticker.mode.get_gangsters())
 			ticker.mode.remove_gangster(target.mind)
-			target.visible_message("<span class='warning'>[src] was destroyed in the process!</span>", "<span class='notice'>You feel a surge of loyalty towards Nanotrasen.</span>")
+			target.visible_message("<span class='warning'>[src] was destroyed in the process!</span>", "<span class='notice'>You feel a sense of peace and security. You are now protected from brainwashing.</span>")
 			removed(target, 1)
 			qdel(src)
 			return -1
 		if(target.mind in ticker.mode.revolutionaries)
 			ticker.mode.remove_revolutionary(target.mind)
 		if((target.mind in ticker.mode.cult) || (target.mind in ticker.mode.blue_deity_prophets|ticker.mode.red_deity_prophets|ticker.mode.red_deity_followers|ticker.mode.blue_deity_followers))
-			target << "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>"
+			target << "<span class='warning'>You feel something try to heal your mind, but you resist it!</span>"
```

Comment:
I feel like this is a bit weird from the perspective of the revhead. Why not "You feel something interfering with your mental conditioning" or something like that. I don't think the brainwashed(may not be canon) revheads consider themselves damaged.